### ISSUE TITLE
have people use inception-resnet-v2 to run the code?

### ISSUE THREAD

**Issue by hsm4703** [davidsandberg/facenet#1057]

**Comment by ntmanh98** [comment]
I tend to run the code using inception-resnet-v2, with train_tripletloss.py. Are you using train_tripletloss.py to train the model?

**Comment by neklom** [comment]
You just change the argument of --model_def to point to the inception_resnet_v2

**Comment by hsm4703** [comment]
you say tend to run the code using inception-resnet-v2, with train_tripletloss.py
you use your image to train  can tell me How many classifications and  How many pictures are in one category
After the final training your  inception-resnet-v2, with train_tripletloss.py   how about performance  better than inception-resnet-v1 ?

**Comment by hsm4703** [comment]
ok i will try it
thanks

neklom <notifications@github.com> 於 2019年7月24日 週三 下午10:06寫道：

**Comment by neklom** [comment]
I don't really understand what you're trying to say I'm sorry, but I think you are referring to training using the triplet loss function, I tried using the tripletloss script and the inception_resnet_v1 architecture but I didn't have a good validation accuracy ! After 350 epochs I had like 86% on the training but nearly 16% on the validation rate ! for the images, I used the Casia-webface dataset.

**Comment by hsm4703** [comment]
I thought you used the database you created.  
After 350 epochs I had like 86% on the training but nearly 16% on the validation rate !  
is not good on accuracy   
do you try add epochs  350 to 400 or more
